src/components/side_panel/chart/line_chart/line_chart_design_panel.xml

```diff
@@ -22,7 +22,7 @@
       </t>
     </GeneralDesignEditor>
     <SeriesWithAxisDesignEditor t-props="props"/>
-    <SidePanelCollapsible isInitiallyCollapsed="true" title.translate="Axes">
+    <SidePanelCollapsible isInitiallyCollapsed="chartSheetIsLocked" title.translate="Axes">
```

Comment:
Shouldn't it be the opposite ? Always expanded when the sheet is locked and the collapsible isn't clickable ?